frontend/src/app/api/auth/session/route.ts

```diff
@@ -2,12 +2,12 @@ import { getSession } from "src/services/auth/session";
 
 import { NextResponse } from "next/server";
 
+export const revalidate = 0;
```

Comment:
we don't want to cache anything here

Reply:
Not sure if this has an effect since a `fetch` isn't called in the session creation AFAICT. Next says it doesn't cache `GET` routes [if cookies](https://nextjs.org/docs/14/app/building-your-application/routing/route-handlers#opting-out-of-caching) are used. [force-dynamic](https://nextjs.org/docs/14/app/building-your-application/routing/route-handlers#convention) might be the way to make sure that the `GET` request is re-rendered every time, but can't hurt to tell it to revalidate as well.

Reply:
Also wondering if this is better as a `POST` method.

Reply:
yeah, I had this in there just for safety, but I just verified and the 'no-store' on the fetch call is doing all the work. We can safely pull this out. As for POST vs GET, I'm not sure I see a reason to make a change, but we can discuss